### PR TITLE
www.annaclemens.com

### PR DESCRIPTION
**Pull request by gork7777**
issue: #79127

### CODE REVIEW

**Comment by wout-codes** [comment]
@gork7777 There is a share button at the below of the article, also two newsletters.

**Comment by gork7777** [comment]
Community and Guide newsletter, right?

**Comment by wout-codes** [comment]
Yeah and a share button

File: AnnoyancesFilter/sections/annoyances.txt [hunk 1]
```diff
@@ -1767,6 +1767,8 @@ revouninstaller.com,logincasino.com,borsagundem.com,officelife.media,kod.ru,mast
 ! ###mc_embed_signup
 kosicednes.sk,tinyprojects.dev,cambio16.com,voxukraine.org,pnas.org,cachevalleydaily.com,bisnis.com,ilpost.it,bsblog.info,alcoparty.com.ua,codastory.com,informazionefiscale.it,kerch.fm,cinemablend.com,prostofinovo.ru,comss.ru,majorgeeks.com,buhgalter.com.ua,providr.com,24tv.ua,babylonbee.com,dexerto.com,football365.com,menrec.com,netdoktor.de,ottawamatters.com,retail-insider.com,walterfootball.com,wweek.com,zozhnik.ru###mc_embed_signup
 !
+annaclemens.com##._form_27
+annaclemens.com##.image-block
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/annoyances.txt:1771]: this rule breaks content. Better remove it.

File: AnnoyancesFilter/sections/annoyances.txt [hunk 2]
```diff
@@ -1767,6 +1767,8 @@ revouninstaller.com,logincasino.com,borsagundem.com,officelife.media,kod.ru,mast
 ! ###mc_embed_signup
 kosicednes.sk,tinyprojects.dev,cambio16.com,voxukraine.org,pnas.org,cachevalleydaily.com,bisnis.com,ilpost.it,bsblog.info,alcoparty.com.ua,codastory.com,informazionefiscale.it,kerch.fm,cinemablend.com,prostofinovo.ru,comss.ru,majorgeeks.com,buhgalter.com.ua,providr.com,24tv.ua,babylonbee.com,dexerto.com,football365.com,menrec.com,netdoktor.de,ottawamatters.com,retail-insider.com,walterfootball.com,wweek.com,zozhnik.ru###mc_embed_signup
 !
+annaclemens.com##._form_27
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/annoyances.txt:1770]: Better block widget `||activehosted.com^$third-party`

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/adservers.txt [hunk 3]
```diff
@@ -5,6 +5,7 @@
 !
 !! https://a.bestcontentitem.top/warp/*?r=
 !
+||activehosted.com^$third-party
```
Comment (Alex-302) [inline EnglishFilter/sections/adservers.txt:8]: no. This is annoyance. Please move to this section:
https://github.com/AdguardTeam/AdguardFilters/blob/d0084bf95ad0616a03e927456c51d25d660134a6/AnnoyancesFilter/sections/annoyances.txt#L1760

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**